outputter/outputter.go

```diff
@@ -22,13 +22,20 @@ type Config struct {
 	Console
 	JSON
 	JSONFormat bool
+	PGSQL      bool
+	Values     map[string]string
 }
 
 // BuildOutputter build new outputter. Depending on the parameters
 // passed will return either a JSON outputter or a Console outputter.
 func BuildOutputter(summary check.Summary, config *Config) Outputter {
-	if (summary.Fail > 0 || summary.Warn > 0 || summary.Pass > 0 || summary.Info > 0) && config.JSONFormat {
-		return NewJSON(config.JSON.Filename)
+	if summary.Fail > 0 || summary.Warn > 0 || summary.Pass > 0 || summary.Info > 0 {
```

Comment:
When you come to merge this I think there is also going to be console output here too

Consider putting the if test inside the specific outputters rather than doing it here, so that you always return a non-nil outputter.